src/main/scala/scalafix/internal/sbt/BlockingCache.scala

```diff
@@ -2,21 +2,32 @@ package scalafix.internal.sbt
 
 import java.{util => ju}
 
+import scala.util.Try
+
 /** A basic thread-safe cache without any eviction. */
 class BlockingCache[K, V] {
 
-  // Number of keys is expected to be very small so the global lock should not be a bottleneck
-  private val underlying = ju.Collections.synchronizedMap(new ju.HashMap[K, V])
+  private val underlying = new ju.concurrent.ConcurrentHashMap[K, V]
```

Comment:
`ConcurrentHashMap` was [ruled out in the past](https://github.com/scalacenter/sbt-scalafix/blob/89cfd04629a6c67d3110e073c1c9948675c85a37/src/main/scala/scalafix/internal/sbt/BlockingCache.scala#L14-L16), but I confirmed that there is a per-key lock on `compute()`.